### DISCUSSION TITLE
Welcome to Invoice_OCR_app Discussions!

### DISCUSSION THREAD

**Discussion by MatKollar** [MatKollar/Invoice_OCR_app#13] [Announcements]
## 👋 Welcome!
If you have any questions, feel free to ask!

**Comment by shivammiyyy** [comment]
well dude i think your project is great but help me for english invoices

**Comment by NeXumZ** [comment]
Hi Team,

thx for sharing this great project with the community. I tried to build the docker images on Ubuntu and run it.
Unfortunately, the homepage with default credentials doesn't work and neither new registration. At this moment, the frond-end part doesn't work for me. Besides, there are some package npm warn incompatibilities among packages.

![Screenshot from 2024-08-20 09-12-46](https://github.com/user-attachments/assets/87cfcc30-f186-4162-b973-bd72548090ca)
